testdata/enableall/util/start_tracer.go

```diff
@@ -0,0 +1,13 @@
+package util
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func TestStartTrace() trace.Span {
```

Comment:
this is another limitation I found with the existing implementation of spancheck's `isSpanStart`func. It will only work with `SelectorExpr` which means the span starting utility functions need to be in a package _different_ from the one being linted. To test that in this function I just made a new `/util` package. But `isSpanStart` should probably be updated to also allow for utility funcs within the same package.

```go
func isSpanStart(info *types.Info, n ast.Node, startSpanMatchers []spanStartMatcher) (spanType, bool) {
	sel, ok := n.(*ast.SelectorExpr)
	if !ok {
		return spanUnset, false
	}
```

Reply:
I did notice this, I agree - it might miss a few calls in places where the tracing packages themselves are misbehaving with spans.